worlds/stardew_valley/__init__.py

```diff
@@ -279,6 +280,7 @@ def fill_slot_data(self) -> Dict[str, Any]:
             "randomized_entrances": self.randomized_entrances,
             "modified_bundles": modified_bundles,
             "client_version": "4.0.0",
+            "mod_versions": mod_versions
```

Comment:
Suggestion: Do not send all mod versions to the client.
Right now, there is a redundancy. We are sending a list of chosen mods, and then a whole dictionary that has all the mods and their version.
Instead, we could send just one item, which is a dictionary of only the chosen mods and their versions. So no mods -> no versions

Reply:
I could send the dictionary.  Just have to rework the client-side to read a dictionary instead.